src/test/index.spec.js

```diff
@@ -0,0 +1,32 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+
+chai.use(chaiHttp);
+
+describe('Testing home endpoint', () => {
+  it('It should return status of 200 on successful page load', done => {
+    const appUrl = '/';
+    chai
+      .request(app)
+      .get(appUrl)
+      .end((error, response) => {
+        expect(response.body).to.be.a('object');
+        expect(response.status).to.equal(200);
+        done();
+      });
+  });
+  it('It should return a message on successful page load', done => {
+    const appUrl = '/';
+    chai
+      .request(app)
+      .get(appUrl)
+      .end((error, response) => {
+        expect(response.body).to.be.a('object');
```

Comment:
`response.body` is always an `object`. This assertion doesn't actually prove that the `api` works as expected.

Reply:
I agree with you. 
That assertion was just to ensure that the unit test was properly set up.
In some use cases, such an assertion can be made in order to ensure that outputs meet requires expectation and acceptance criteria.

Reply:
the second assertion is useful. The first is not.

Remove it or change it to something some more explicit, like the properties expected on the response body.

Reply:
I've made the necessary changes to the most recent comment raised about the first assertion.

Reply:
I have made changes to the comments you raised on the first assertion.